Sentaur.Leaderboard.Api/User.cs

```diff
@@ -0,0 +1,7 @@
+namespace Sentaur.Leaderboard.Api;
+
+public class User
+{
+    public string UserName { get; set; }
```

Comment:
username is one word

```suggestion
    public string Username { get; set; }
```